specification/azurefleet/resource-manager/Microsoft.AzureFleet/stable/2024-11-01/azurefleet.json

```diff
@@ -0,0 +1,3581 @@
+          "description": "Maximum value. Double.MaxValue(1.7976931348623157E+308)",
+          "minimum": 0
+        }
+      }
+    },
+    "VMAttributeMinMaxInteger": {
+      "type": "object",
+      "description": "While retrieving VMSizes from CRS, Min = 0 (uint.MinValue) if not specified, Max = 4294967295 (uint.MaxValue) if not specified. This allows to filter VMAttributes on all available VMSizes.",
+      "properties": {
+        "min": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Min VMSize from CRS, Min = 0 (uint.MinValue) if not specified.",
+          "minimum": 0
+        },
+        "max": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Max VMSize from CRS, Max = 4294967295 (uint.MaxValue) if not specified.",
+          "minimum": 0
+        }
+      }
+    },
+    "VMAttributeSupport": {
+      "type": "string",
+      "description": "VMSizes supported by Azure VMs. Included is a union of Excluded and Required.",
+      "enum": [
+        "Excluded",
+        "Included",
+        "Required"
+      ],
+      "x-ms-enum": {
+        "name": "VMAttributeSupport",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "Excluded",
+            "value": "Excluded",
+            "description": "All VMSizes having the feature support will be excluded."
+          },
+          {
+            "name": "Included",
+            "value": "Included",
+            "description": " VMSizes that have the feature support and that do not have the feature support will be used. Included is a union of Excluded and Required."
+          },
+          {
+            "name": "Required",
+            "value": "Required",
+            "description": "Only the VMSizes having the feature support will be used."
+          }
+        ]
+      }
+    },
+    "VMAttributes": {
+      "type": "object",
+      "description": "VMAttributes that will be used to filter VMSizes which will be used to build Fleet.",
+      "properties": {
+        "vCpuCount": {
+          "$ref": "#/definitions/VMAttributeMinMaxInteger",
+          "description": "The range of vCpuCount specified from Min to Max. Must be specified if VMAttributes are specified, either Min or Max is required if specified."
+        },
+        "memoryInGiB": {
+          "$ref": "#/definitions/VMAttributeMinMaxDouble",
+          "description": "The range of memory specified from Min to Max. Must be specified if VMAttributes are specified, either Min or Max is required if specified."
+        },
+        "localStorageSupport": {
+          "$ref": "#/definitions/VMAttributeSupport",
+          "description": "Specifies whether the VMSize supporting local storage should be used to build Fleet or not."
+        },
+        "localStorageInGiB": {
+          "$ref": "#/definitions/VMAttributeMinMaxDouble",
+          "description": "The range of local storage in GB specified from Min to Max."
```

Comment:
LocalStorageSupport should be set to \"Included\" or \"Required\" to use this VMAttribute. \nIf localStorageSupport is \"Excluded\", this VMAttribute can not be used." should be added to the description for localStorageinGiB.

Reply:
Updated.